docker-compose.yml

```diff
@@ -8,7 +8,34 @@ services:
     ports:
       - "8081:8080"
     environment:
```

Comment:
What about defining both services (keycloak,redhat-sso) and pass the service name as argument to `docker-compose up` . As alternative solution I propose two `docker.compose.yml` files but maybe overkill